modules/config-updates-38.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: CONCEPT
+[id="config-updates-37"]
+== Configuration updates for Quay 3.8
+
+[id="new-configuration-fields-38"]
+=== New configuration fields
+
+The following configuration fields have been introduced with {productname} 3.8: 
+
+[options="header"]
+|=== 
+
+|Field | Type |Description 
+| **FEATURE_LISTEN_IP_VERSION** | String | Enables IPv6, IPv6, or dual-stack protocol family. This configuration field must be properly set, otherwise {productname} fails to starts. 
```

Comment:
There are two IPv6 here,  Is this a typo?  What we want to say is `IPv4, IPv6 , or dual-stack protocol family` ?